src/legacy/core_plugins/visualizations/public/np_ready/types/types_service.ts

```diff
@@ -42,13 +46,23 @@ export class TypesService {
     };
   }
 
+  public start({ VisTypesRegistryProvider, Private }: StartDependencies) {
+    const visTypes = Private(VisTypesRegistryProvider);
+    return {
+      get: (visualization: string) => {
+        return visTypes.get(visualization);
```

Comment:
shouldn't this call use `.byName[visualization]`? I don't see an implementation of the `get` function on indexed array